collectors/ciscomeraki/collector.js

```diff
@@ -165,7 +165,7 @@ class CiscomerakiCollector extends PawsCollector {
     }
 
     handleThrottlingError(error, state, callback) {
-        const maxRandom = 5;
+        const maxRandom = 5 || parseInt(process.env.ciscomeraki_throttlingmaxrandom, 10);
```

Comment:
won't this always result in `maxRandom = 5`?

Reply:
my mistake :) resolved